app/models/cloud_tenant.rb

```diff
@@ -7,12 +7,14 @@ class CloudTenant < ActiveRecord::Base
   belongs_to :ext_management_system, :foreign_key => "ems_id", :class_name => "ManageIQ::Providers::CloudManager"
   has_many   :security_groups
   has_many   :cloud_networks
+  has_many   :cloud_subnets
+  has_many   :network_ports
+  has_many   :network_routers
   has_many   :shared_cloud_networks, -> { where :shared => true }, :through => :ext_management_system, :source => :cloud_networks
   has_many   :vms
   has_many   :vms_and_templates
   has_many   :miq_templates
   has_many   :floating_ips
-  has_many   :cloud_resource_quotas
```

Comment:
Why was this deleted?

I don't see mention of it anywhere else (like if it were moved or something)...